lib/collections/schemas/accounts.js

```diff
@@ -217,6 +217,12 @@ export const Accounts = new SimpleSchema({
       }
     },
     optional: true
+  },
+  walletBalance: {
+    type: Number,
+    decimal: true,
+    defaultValue: 0,
+    optional: true
   }
```

Comment:
You directly modify the accounts schema here. See comments above

Reply:
I have removed the extension and stuck with modifying the accounts schema directly. Extending it in my plugin was not working as expected.